vendor/assets/javascripts/brainstem/utils.coffee

```diff
@@ -55,6 +55,8 @@ class window.Brainstem.Utils
 
   @wrapError = (collection, options) ->
     error = options.error
-    options.error = (response) ->
-      error(collection, response, options) if error
-      collection.trigger('error', collection, response, options)
+    # only wrap if we have a handler
+    if error
```

Comment:
If we check first, we won't trigger an error event unless the user passes an error in. The way it was before, it would only invoke the passed in error if it exists and always trigger an error event. It's also how Backbone does it http://backbonejs.org/docs/backbone.html#section-255